buildingmotif-app/src/environments/environment.ts

```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  API_URL: "http://localhost:5000",
```

Comment:
Is this being read from environment variables?

Reply:
I don't believe so, it's an angular convention. https://v17.angular.io/guide/build#configuring-application-environments